definitions/features/service.rb

```diff
@@ -53,6 +53,16 @@ def action_past_tense(action)
     action_word_modified(action) + 'ed'
   end
 
+  def perform_action_on_service(action, service)
+    command = service_command(action, service)
+    if action == 'status'
+      status = execute(command)
+      puts "\n\n#{status}\n\n"
```

Comment:
We should not do printing from features, but rather returning the status as return value and printing from procedure, as there is the reporter that should be used for reporting the data to the user.

Reply:
On the other hand, I see that the change was already in, so leaving this as optional for this PR to change that.